frontend/src/components/Displays/ConfirmScheduleDialogs/LocalizationVerification/ConfirmLocalizationDialog.tsx

```diff
@@ -44,7 +44,6 @@ export const ConfirmLocalizationDialog = ({
                     )}
                     <HorizontalContent>
                         <Checkbox
-                            crossOrigin={undefined}
```

Comment:
Caused a compile error after the update. @Eddasol Is it ok to remove this?

Reply:
It never did anything, but needed to have it since it was required in the previous version. This is better!